app/controllers/projects_controller.rb

```diff
@@ -196,6 +196,16 @@ def delete_episode
     redirect_to project_path(nil, @project), notice: "Removed hackweek #{@episode.name}"
   end
 
+  def follow
```

Comment:
The actions `follow` and `unfollow`, sounds me that they have to be in a specific controller with CRUD actions.